go.mod

```diff
@@ -6,10 +6,12 @@ require (
 	github.com/dlclark/regexp2 v1.2.0
 	github.com/dop251/goja_nodejs v0.0.0-20200811150831-9bc458b4bbeb
 	github.com/go-sourcemap/sourcemap v2.1.3+incompatible
-	github.com/golangci/golangci-lint v1.33.0 // indirect
 	github.com/google/go-cmp v0.5.2
-	github.com/tsedgwick/goja v0.0.3 // indirect
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
Are these changes expected?

Reply:
yes this was the result of `go mod tidy` so they're expected.